src/posts/published/how-to-use-usereducer-as-a-finite-state-machine/index.mdx

```diff
@@ -20,11 +20,11 @@ relevantCourseNickname: 'xstate'
 import { FootnoteMarker as Marker } from '../../../components/Footnotes'
 import LightBulb from './LightBulb'
 
-I recently attempted to get XState into the Webflow codebase to manage some challenging and complex UIs but was met with resistance. No big deal, this is a part of work, consensus driving, disagreeing and commiting, etc. If it hasn't happened to you yet, it will.
+I recently attempted to get XState into the Webflow codebase to manage some challenging and complex UIs but was met with resistance. No big deal, this is a part of work, consensus driving, disagreeing and committing, etc. If it hasn't happened to you yet, it will.
 
-One of the things you need when that happens is a good imagination. How do we make the tools we already have do the things we want in a reasonable way? I'm going to help you with that today by showing you a way to make `useReducer` behave like a simple finite state machine in React.
+One of the things you need when that happens is a good imagination. How do we make the tools we already have to do the things we want in a reasonable way? I'm going to help you with that today by showing you a way to make `useReducer` behave like a simple finite state machine in React.
```

Comment:
This isn't right, the original is correct. The "tools" are "doing" things.

Reply:
Oops, you are correct. I read that completely wrong in my head, my bad.